### ISSUE TITLE
Create a home page for the platform 

### ISSUE THREAD

**Issue by nikhilbghodke** [The-Robotics-Forum/sampark#5]
**Is your feature request related to a problem? Please describe.**
Sampark does not have a page which lists all the features of the platform some small section for the contributors.

**What features does our page has? **
we currently do not have a homepage itself..

**What features do we want extra?**
1. The homepage should list all the features of the platform with appropriate graphics. You can find about all the features in the readme file of this repo.
2. There should be a section for contributor and 4-5 lines about their background and social media links.
3. A good loading animation with smapark logo would be great to have, sampark logo can be found inside the readme itself.
4. In the navbar for home page user should be able to login or signup, or if he is already logined then link to move to `/rooms` page.
5. Make sure your page uses this color scheme [https://colorhunt.co/palette/2763](https://colorhunt.co/palette/2763)

**How should you submit a pull request for this issue**
1. Create a '/' route and add your page there.
2. Send a pull request as described in readme for frontend

**any tips for solving this issue**
Make sure the page is responsive , You would need knowledge of react and react-router-dom for completion of this issue.

**Comment by Abhir-24** [comment]
I can give a try on this issue under SWOC

**Comment by nikhilbghodke** [comment]
Hi @Abhir-24,
you can surely start working on this, just go through all features that the application has as we would like to have all of them noted on our homepage.
Happy learning :)

**Comment by Abhir-24** [comment]
The deployed app link given is not working

**Comment by nikhilbghodke** [comment]
yes, we are aware about it, heroku free dynos for this month for our account have finished, we hope to get free dynos for next month tomorrow

Till the time you can run the application locally

**Comment by nikhilbghodke** [comment]
hey @Abhir-24 , deployed app is up now.

**Comment by Abhir-24** [comment]
@nikhilbghodke can u brief me about point 2 of the extra features mentioned as I am not able to understand what background and social media details of the user are being talked about?

**Comment by nikhilbghodke** [comment]
yeah, we just want to show few lines about the contributors and social media links like instagram,linkedin and github for each contributor on our homepage
